modules/standard/Path.chpl

```diff
@@ -527,16 +555,23 @@ private proc joinPath(paths: [] string): string {
 }
 
 // Normalize leading slash count to a value between 0 and 2.
-private proc normalizeLeadingSlashCount(name: string): int {
-  var result = if name.startsWith(pathSep) then 1 else 0;
+private proc normalizeLeadingSlashCount(path: string): int {
+  var result = if path.startsWith(pathSep) then 1 else 0;
 
   // Two leading slashes has a special meaning in POSIX.
-  if name.startsWith(pathSep * 2) && !name.startsWith(pathSep * 3) then
+  if path.startsWith(pathSep * 2) && !path.startsWith(pathSep * 3) then
     result = 2;
 
   return result;
 }
 
+pragma "no doc"
+pragma "last resort"
+private proc normalizeLeadingSlashCount(name: string): int {
+  compilerWarning("Path.normalizeLeadingSlashCount: Argument 'name' is deprecated - use 'path' instead");
+  return normalizeLeadingSlashCount(path=name);
+}
+
```

Comment:
No need to change anything here, but I am only realizing now that this was actually a private function. These are not user-facing, and we can change them more freely, and without doing real deprecation at all. No harm to have the deprecated version, either, though